src/Adapter/AbstractObjectModelRepository.php

```diff
@@ -95,6 +116,43 @@ protected function getObjectModel(int $id, string $objectModelClass, string $exc
         return $objectModel;
     }
 
+    /**
+     * @param int $id
+     * @param string $objectModelClass
```

Comment:
```suggestion
     * @param class-name<ObjectModel> $objectModelClass
```

see https://phpstan.org/writing-php-code/phpdoc-types#class-string

Reply:
The parameter should probably be renamed to `$objectModelClassName`

Reply:
It should be `class-string<ObjectModel>` and not `class-name` :thinking: